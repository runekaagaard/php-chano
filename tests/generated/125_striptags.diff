001- some html with alert("You smell") disallowed  tags
001+ some html with alert(&quot;You smell&quot;) disallowed  tags