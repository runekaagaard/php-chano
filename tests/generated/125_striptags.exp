some html with alert("You smell") disallowed  tags