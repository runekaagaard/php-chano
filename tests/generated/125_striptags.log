
---- EXPECTED OUTPUT
some html with alert("You smell") disallowed  tags
---- ACTUAL OUTPUT
some html with alert(&quot;You smell&quot;) disallowed  tags
---- FAILED
