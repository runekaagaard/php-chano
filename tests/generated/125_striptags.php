<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'some <b>html</b> with <script>alert("You smell")</script> disallowed <img /> tags')));
foreach ($items as $i) echo $i->input->striptags();
