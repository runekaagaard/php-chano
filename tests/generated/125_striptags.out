some html with alert(&quot;You smell&quot;) disallowed  tags